
---
string path_to_image
